Automattic/wp-calypso
# Fix Jetpack product card disabled message not showing.

#### Changes proposed in this Pull Request

Related to: 1164141197617539-as-1201955329670015

Many Jetpack products are not compatible with multi-site WordPress sites and therefore these products show up as disabled on the pricing page, however the disabled message (The reason it is disabled) is not displaying.  This PR fixes this and now displays a message when a product card is disabled.
- Also fixed the product price not showing as disabled.

#### Testing instructions

1. Spin up a new Jurassic Ninja **multi-site install**.  ( click **Want more options**? -> **Launch Multisite on subdirs** ).
2. Connect Jetpack.
3. Go to `https://cloud.jetpack.com/pricing/:site`, (replace `:site` with the site slug of your JN multisite site created above)
4. See how the disabled products do not display any message saying why they are disabled.  See how the product prices are not also disabled. (See screenshot below)
5. Now checkout this PR and spin it up in Calypso blue. (`yarn start`) and Calypso green concurrently. (`yarn start-jetpack-cloud-p`) )
6. Go to `http://jetpack.cloud.localhost:3000/pricing/:site` (replace `:site` with the site slug of your JN multisite site created above)
7. Verify that the disabled reason message is now showing on all disabled product cards. (See screenshot below)
8. Verify that the product price is also disabled.
9. Go to `http://calypso.localhost:3000/jetpack/connect/plans/:site`
10. Check steps 7 & 8.
11. Go to `http://calypso.localhost:3000/plans/:site`
12. Check steps 7 & 8.

#### Screenshots

**BEFORE:**

![Markup 2022-03-14 at 10 05 06](https://user-images.githubusercontent.com/11078128/158198566-7b8acf6e-f559-4d9b-af18-bfb7d3d4c265.png)

**AFTER:**

![Markup 2022-03-14 at 10 07 04](https://user-images.githubusercontent.com/11078128/158198617-b1403b99-c803-4369-b1c8-2c5620c33675.png)



*
